lib/ParseSIL/ParseSIL.cpp

```diff
@@ -2767,6 +2838,83 @@ bool SILParser::parseSILInstruction(SILBuilder &B) {
     ResultVal = B.createBuiltin(InstLoc, Id, ResultTy, subs, Args);
     break;
   }
+  // SWIFT_ENABLE_TENSORFLOW
+  case SILInstructionKind::GraphOperationInst: {
+    // Parse graph operation name.
+    if (P.Tok.isNot(tok::string_literal)) {
+      P.diagnose(P.Tok, diag::expected_tok_in_sil_instr, "graph_op name");
+      return true;
+    }
+    StringRef rawString = P.Tok.getText().drop_front().drop_back();
+    Identifier name = P.Context.getIdentifier(rawString);
+    P.consumeToken(tok::string_literal);
+
+    // Parse graph operation arguments.
+    if (P.Tok.isNot(tok::l_paren)) {
+      P.diagnose(P.Tok, diag::expected_tok_in_sil_instr, "(");
+      return true;
+    }
+    SmallVector<SILValue, 4> arguments;
+    P.consumeToken(tok::l_paren);
+    while (true) {
+      if (P.consumeIf(tok::r_paren))
```

Comment:
Same thing about the comma separated list thing.

Reply:
Done.